### PR TITLE
Added front-end technical test

### PR DESCRIPTION
**Pull request by jack-mullen**
@tyronwyt Would you mind having a look at this when you have a chance? 

Ping me on slack if you've got any specific issues or questions about the implementation.

### CODE REVIEW

File: front-end/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "@types/react-dom": "^18.2.7",
+    "concurrently": "^8.2.0",
+    "json-server": "^0.17.3",
+    "parse-link-header": "^2.0.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "typescript": "^4.9.5",
+    "web-vitals": "^2.1.4"
+  },
```
Comment (tyronwyt) [inline front-end/package.json:21]: Fix dependency warning when running `npm run start`
```suggestion
  },
    "devDependencies": {
    "@babel/plugin-proposal-private-property-in-object": "7.21.11"
  },
```

**Review by tyronwyt — APPROVED** [review]
Looks good, ran first time following instructions, API links work, two small comments below:

**Review by tyronwyt — APPROVED** [review]
Looks good, ran first time following instructions, API links work, two small comments  above:

**Comment by jack-mullen** [comment]
@tyronwyt Thanks! Have implemented both of your suggestions

**Pull request merged**